packages/docs/docs/charting/capturing-vital-signs.md

```diff
@@ -0,0 +1,136 @@
+# Capturing Vital Signs
+
+Vital signs are a group of important medical signs that measure the body's most vital (life-sustaining) functions. The four main vital signs are body temperature, pulse rate, respiration rate, and blood pressure. However, sometimes other measures, such as blood oxygen, weight, and blood glucose level, are included as well.
+
+## The `Obeservation` Resource
+
+Vital signs are stored as [`Observation`](/docs/api/fhir/resources/observation) resources. To track the specific vital that is being observed, use the `Observation.code` element. This describes what was observed, and is sometimes called the observation &quot;name&quot;. This should be coded using [LOINC](https://loinc.org/search) codes per US core guidelines.
+
+To record the actual value of a measurement, you can use one of the various value fields on the `Observation` resource. There are fields for `valueQuantity`, `valueString`, `valueBoolean` and more. To see all of the available fields on the `Observation` resource, see the [`Observation` resource reference](docs/api/fhir/resources/observation).
+
+**Example: **
+
+```ts
+
+{
+  resourceType: 'Observation',
+  id: 'example-observation-1',
+  // ...
+  code: {
+    system: 'http://loinc.org',
+    code: '8310-5',
+    display: 'Body temperature'
+  },
+  valueQuantity: {
+    value: 98.2,
+    unit: 'degrees fahrenheit'
```

Comment:
Is this ValueQuantity correct?  Samples look more like this 
![image](https://github.com/medplum/medplum/assets/2314326/b7051729-d049-4d3f-b616-9906ba14fe1b)